data/brands/amenity/restaurant.json

```diff
@@ -45,6 +45,25 @@
     }
   },
   "items": [
+    {
+      "displayName": "Bar-B-Q Plaza",
+      "locationSet": {"include": ["th"]},
+      "matchNames": [
+        "BBQ Plaza",
+        "Bar.B.Q Plaza"
```

Comment:
NSI already recognizes these two spelling variants, because it strips punctuation and lowercases before comparing names.

Reply:
Understood how Bar.B.Q Plaza will already be recognized without punctuation. But How is BBQ Plaza recognized? It would need to strip the "ar" in "Bar" as well.